spec/core_functions/color/hsl/error/zero_args.hrx

```diff
@@ -0,0 +1,12 @@
+<===> input.scss
+zero-args {
```

Comment:
Even though the tests around this predate [the style guide](https://github.com/sass/sass-spec/blob/master/STYLE_GUIDE.md), these new tests should follow it.

Reply:
Done. I know it's frowned upon to impose new style on old code, but I saw enough low-hanging fruit to justify cleaning up the surrounding tests too.